code/ui/manager/src/components/upgrade/UpgradeBlock.tsx

```diff
@@ -40,10 +41,13 @@ const Container = styled.div(({ theme }) => ({
   borderRadius: 5,
   padding: 20,
   marginTop: 0,
-  maxWidth: 400,
   borderColor: theme.appBorderColor,
   fontSize: theme.typography.size.s2,
   width: '100%',
+
+  [`@media ${BREAKPOINT_MIN_600}`]: {
```

Comment:
Should we have a constant that is this whole string, like `IS_MOBILE_BREAKPOINT`?

Reply:
I chose to do:
```tsx
export const BREAKPOINT = 600;
export const MEDIA_MIN_BREAKPOINT = `@media (min-width: ${BREAKPOINT}px)`;
```

And in the layout provider:
```tsx
const isDesktop = useMediaQuery(`(min-width: ${BREAKPOINT}px)`);
const isMobile = !isDesktop;
```

Reply:
Change to `MEDIA_DESKTOP_BREAKPOINT`

Reply:
Made the c hange